debian/bionic/foreman/foreman-dynflow-sidekiq.install

```diff
@@ -0,0 +1,4 @@
+extras/systemd/dynflow-sidekiq@.service /lib/systemd/system
+bundler.d/dynflow_sidekiq.rb /usr/share/foreman/bundler.d
+config/dynflow/orchestrator.yml etc/foreman
```

Comment:
Shouldn't this be `etc/foreman/dynflow` (same below):
```suggestion
config/dynflow/orchestrator.yml etc/foreman/dynflow
```

Reply:
It definitely should :( done.